manifests/storage.yaml

```diff
@@ -13,36 +13,25 @@ spec:
   selector:
     matchLabels:
       k8s-app: hostpath-provisioner
-
   template:
     metadata:
       labels:
         k8s-app: hostpath-provisioner
-
     spec:
       containers:
         - name: hostpath-provisioner
-          image: kjackal/hostpath-provisioner:latest
+          image: balchu/hostpath-provisioner:1.1.1
```

Comment:
let's revert this as well. even better, use the image from `cdkbot/hostpath-provisioner`